plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/MetadataEntry.java

```diff
@@ -173,11 +177,19 @@ public Optional<Boolean> isChangeDataFeedEnabled()
         return Optional.of(changeDataFeedEnabled);
     }
 
-    public static Map<String, String> configurationForNewTable(Optional<Long> checkpointInterval, Optional<Boolean> changeDataFeedEnabled)
+    public static Map<String, String> configurationForNewTable(
+            Optional<Long> checkpointInterval,
+            Optional<Boolean> changeDataFeedEnabled,
+            ColumnMappingMode columnMappingMode,
+            OptionalInt maxFieldId)
     {
         ImmutableMap.Builder<String, String> configurationMapBuilder = ImmutableMap.builder();
         checkpointInterval.ifPresent(interval -> configurationMapBuilder.put(DELTA_CHECKPOINT_INTERVAL_PROPERTY, String.valueOf(interval)));
         changeDataFeedEnabled.ifPresent(enabled -> configurationMapBuilder.put(DELTA_CHANGE_DATA_FEED_ENABLED_PROPERTY, String.valueOf(enabled)));
+        if (columnMappingMode == ColumnMappingMode.ID || columnMappingMode == ColumnMappingMode.NAME) {
```

Comment:
```
switch (columnMappingMode) {
    case NONE -> {}
    case ID, NAME -> {
        ...
    }
    case UNKNOWN -> throw new UnsupportedOperationException();
}
```

Reply:
`case NONE -> {}` causes checkstyle error "WhitespaceAround: '{' is not followed by whitespace". Changed to `case NONE -> { /* do nothing */ }`.